immich.json

```
@@ -0,0 +1,119 @@
+{
+  "Immich": {
+    "description": "Immich is a high-performance self-hosted photo and video backup solution with official mobile apps and 3rd-party smart TV apps. <p>Note: some optional features included in this Rockon rely on external services that may require additional accounts. These services are transparently listed and configurable during the initial WebUI setup.</p> <p>Includes multiple custom docker images: Immich server, Immich machine-learning server, Redis and Postgres.</p> <p>Based on the official Docker Compose <a href='https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml' target='_blank'>configuration file</a>, available for amd64 and arm64 architecture.</p>",
```

```suggestion
    "description": "Immich is a high-performance self-hosted photo and video backup solution with official mobile apps and 3rd-party smart TV apps. <p>Note: some optional features included in this Rock-on rely on external services that may require additional accounts. These services are transparently listed and configurable during the initial WebUI setup.</p> <p>Includes multiple custom docker images: <a href='https://github.com/immich-app/immich/pkgs/container/immich-server' target='_blank'>Immich server</a>, <a href='https://github.com/immich-app/immich/pkgs/container/immich-machine-learning' target='_blank'>Immich machine-learning server</a>, <a href='https://hub.docker.com/r/valkey/valkey' target='_blank'>Redis</a> and <a href='https://github.com/immich-app/base-images/pkgs/container/postgres' target='_blank'>Postgres</a>.</p> <p>Based on the official Docker Compose <a href='https://github.com/immich-app/immich/releases/latest/download/docker-compose.yml' target='_blank'>configuration file</a>, available for amd64 and arm64 architecture.</p>",
```
cosmetic suggestions to include the custom docker container links as well and move from Rockon to Rock-on.